qemu/tests/cfg/qemu_img_convert_image_with_unaligned_size.cfg

```diff
@@ -0,0 +1,17 @@
+- qemu_img_convert_image_with_unaligned_size:
+    only raw
+    virt_test_type = qemu
+    type = qemu_img_convert_image_with_unaligned_size
+    start_vm = no
+    required_qemu = [2.12.0-83,)
```

Comment:
What about qemu-kvm-rhev, not in the scope of testing? like 2.12.0-38

Reply:
Yes, not in the scope. The related bug[1] is closed by our PM.

[1]https://bugzilla.redhat.com/show_bug.cgi?id=1649788